torch_ort/docker/Dockerfile.ort-default-stable-torch1.12.1-onnxruntime190-cu116-cudnn8-devel-ubuntu18.04

```diff
@@ -3,39 +3,43 @@
 # Licensed under the MIT License.
 
 # CUDA development image for building sources
-FROM nvidia/cuda:10.2-cudnn8-devel-ubuntu18.04 as builder
-
-ARG TORCH_CUDA_VERSION=cu102
-ARG TORCH_VERSION=1.8.1
-ARG TORCHVISION_VERSION=0.9.1
+FROM nvidia/cuda:11.6.2-cudnn8-devel-ubuntu18.04 as builder
+ARG TORCH_CUDA_VERSION=cu116
+ARG ONNXRUNTIME_TRAINING_VERSION=1.12.0
+ARG TORCH_VERSION=1.12.1
+ARG TORCHVISION_VERSION=0.13.1
 
 # Install and update tools to minimize security vulnerabilities
 RUN apt-get update
 RUN apt-get install -y software-properties-common wget apt-utils patchelf git libprotobuf-dev protobuf-compiler cmake
 RUN unattended-upgrade
 RUN apt-get autoremove -y
 
+# install miniconda (comes with python 3.9 default)
+RUN apt-get install curl
+ARG CONDA_VERSION=py39_4.10.3
+ARG CONDA_URL=https://repo.anaconda.com/miniconda/Miniconda3-${CONDA_VERSION}-Linux-x86_64.sh
+RUN curl -fSsL --insecure ${CONDA_URL} -o install-conda.sh &&\
+    /bin/bash ./install-conda.sh -b -p /opt/conda &&\
```

Comment:
Can we instead do a local installation (not in /opt/conda)? You can refer to how it is done in [orrttraining](https://github.com/microsoft/onnxruntime/blob/main/tools/ci_build/github/linux/docker/Dockerfile.ubuntu_gpu_training#L37-L46).